src/app/shared/toggler/toggler.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TogglerComponent } from './toggler.component';
+
+
+
+@NgModule({
+  declarations: [
+    TogglerComponent
+  ],
+  imports: [
+    CommonModule
+  ]
+})
```

Comment:
add TogglerComponent to exports: [] as well to use this component outside of current module